accounting-service/app/src/main/java/com/microservices/accounting/dto/RevertPaymentDto.java

```diff
@@ -0,0 +1,18 @@
+package com.microservices.accounting.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class RevertPaymentDto {
```

Comment:
Actually I think the only needed parameter for reverting is payment id?

Reply:
Ok, I agree. Could you fix it in a diagram later?

Reply:
Of course